### PR TITLE
Moved hardcoded URLs to a constant file Fixed #2850

### LINKED ISSUE #2850: Moving hardcoded URLs from About Activity to a constant file
**Summary:** 
There are many hardcoded strings which makes them difficult to locate and change the URL in case needed, so I think we should move them build.gradle file.

like
```
@OnClick(R.id.about_credits)
    public void launchCredits(View view) {
        Utils.handleWebUrl(this,Uri.parse("https://github.com/commons-app/apps-android-commons/blob/master/CREDITS/\\"));
    }

    @OnClick(R.id.about_privacy_policy)
    public void launchPrivacyPolicy(View view) {
        Utils.handleWebUrl(this,Uri.parse("https://github.com/commons-app/apps-android-commons/wiki/Privacy-policy\\"));
    }


    @OnClick(R.id.about_faq)
    public void launchFrequentlyAskedQuesions(View view) {
        Utils.handleWebUrl(this,Uri.parse("https://github.com/commons-app/apps-android-commons/wiki/Frequently-Asked-Questions\\"));
    }
```

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by madhurgupta10**
**Description**
Moved the hardcoded URLs to a constant file so that the code is easier to maintain, also it removed the duplicate strings :)

**Fixes #2850 Moving URLs from About Activity to build.gradle**

**Tests performed**
Tested betaDebug on Xiaomi Mi A1 with API level 28 stock android.

### CODE REVIEW

**Comment by madhurgupta10** [comment]
@maskaravivek changes made as per the [discussion](https://github.com/commons-app/apps-android-commons/issues/2850#issuecomment-479871079)

File: app/src/main/java/fr/free/nrw/commons/Urls.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package fr.free.nrw.commons;
+
+class Urls {
+    static final String NEW_ISSUE_URL = "https://github.com/commons-app/apps-android-commons/issues";
+    static final String GITHUB_REPO_URL = "https://github.com/commons-app/apps-android-commons";
+    static final String WEBSITE_URL = "https://commons-app.github.io";
+    static final String CREDITS_URL = "https://github.com/commons-app/apps-android-commons/blob/master/CREDITS";
+    static final String FAQ_URL = "https://github.com/commons-app/apps-android-commons/wiki/Frequently-Asked-Questions";
+    static final String PLAY_STORE_URL = "https://play.google.com/store/apps/details?id=fr.free.nrw.commons";
+    static final String TRANSLATE_WIKI_URL = "https://translatewiki.net/w/i.php?title=Special:Translate&group=commons-android-public static final Strings&filter=%21translated&action=translate&language=";
```
Comment (neslihanturan) [inline app/src/main/java/fr/free/nrw/commons/Urls.java]: This string seems problematic, can you verify @madhurgupta10 ?

Comment (madhurgupta10) [reply app/src/main/java/fr/free/nrw/commons/Urls.java]: @neslihanturan Fixed. Sorry for the Typo, I used "find and replace" from the android studio and did not notice, this also got changed.

Comment (neslihanturan) [reply app/src/main/java/fr/free/nrw/commons/Urls.java]: Find and replace is quite risky:)

**Pull request merged**